pkg/ovs/ovs.go

```diff
@@ -40,18 +25,24 @@ func (tx *Transaction) exec(cmd string, args ...string) (string, error) {
 		return "", tx.err
 	}
 
+	cmdpath, err := exec.LookPath(cmd)
```

Comment:
Yeah, we could do that. It looks like everywhere else in openshift/kubernetes doesn't really worry about caching path lookups though...